nesta/packages/dq/text.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import itertools
+from collections import Counter
+##string length
+def string_length(input):#, column):

```

Comment:
Maybe include an argument that allows the user to choose whether to count NaN values. If triggered you could do `fillna('')` on the data